regolith/helpers/u_logurlhelper.py

```diff
@@ -52,16 +53,39 @@ def construct_global_ctx(self):
 
     def db_updater(self):
         rc = self.rc
-        key = f"{rc.name}"
+        key = f"{rc.projectum_id}"
         filterid = {'_id' : key}
-        projectum = rc.client.find_one(rc.database, rc.coll, filterid)
+        found_projectum = rc.client.find_one(rc.database, rc.coll, filterid)
+        # find all similar projectum ids
+        ids = []
+        for projectum in self.gtx["projecta"]:
```

Comment:
what is happening here?  can we just use ``fragment_retrieval()`` and get a list of all projecta that match a pattern?

Reply:
Is it okay if I leave this part in? I just tried to use fragment retrieval but I can't seem to get it to work...

Reply:
what exactly  isn't working?  We can put a PR in to fix fragment retrieval if it is not working.  But I think that
```
pra = fragment_retrieval(self.gtx["projecta"], ["_id"], rc.projectum_id)
```
should just give you the list you are after